recipe/meta.yaml

```diff
@@ -30,9 +28,7 @@ requirements:
     - python
     - python-dateutil >=2.6
     - python-tzdata >=2020.1
-    - backports.zoneinfo >=0.2.1  # [py<39]
     - time-machine >=2.6.0
```

Comment:
It's only required for testing https://github.com/python-pendulum/pendulum/blob/3.1.0/pyproject.toml#L27 and https://github.com/search?q=repo%3Apython-pendulum%2Fpendulum%20%22time-machine%22&type=code

Reply:
If I'm correct, I think @Jrice1317 was talking about `backports.zoneinfo` instead of `time-machine`